circuits/verify_nullifier.circom

```diff
@@ -59,15 +63,15 @@ template plume_v1(n, k, message_length) {
     g[0] = get_genx(n, k);
     g[1] = get_geny(n, k);
     c_sha256.preimage_bit_length <== sha256_preimage_bit_length;
+    c_sha256.pk_compressed <== pk_compressor.compressed;
 
     for (var i = 0; i < 2; i++) {
         for (var j = 0; j < k; j++) {
             c_sha256.coordinates[i][j] <== g[i][j];
-            c_sha256.coordinates[2+i][j] <== pk[i][j];
```

Comment:
Is it expected as removed line?

Reply:
Yes, pk is passed as input right before in compressed form